RELEASE/scripts/autoscend/quests/level_11.ash

```diff
@@ -2463,6 +2477,7 @@ boolean L11_ronCopperhead()
 			auto_log_info("Bringing the Camel to spit on a Red Butler for glark cables.");
 			handleFamiliar($familiar[Melodramedary]);
 		}
+		//set_property("auto_nextEncounter","Ron \"The Weasel\" Copperhead");	//this encounter is technically predictable, but mafia does not track progress?
```

Comment:
Not looking at code ATM. We have a function to determine how much delay is needed in zones. Could this be used to predict shen is next if delay is zero?

Reply:
not exactly because only some of the location monsters advance delay

Reply:
Good point. I think this would be worth a mafia forum post for a feature request

Reply:
Shen also has variable delay of 5-7 per encounter.